website/docs/r/dns_record_set.markdown

```diff
@@ -34,15 +44,25 @@ resource "google_dns_managed_zone" "prod" {
   name     = "prod-zone"
   dns_name = "prod.mydomain.com."
 }
+```
 
-resource "google_dns_record_set" "frontend" {
+### Adding a SPF record
+
+`\"` must be added around your `rrdatas` for a SPF record. Otherwise `rrdatas` string gets splitted on spaces.
```

Comment:
SPF is a TXT field. The other fields like A, CNAME, MX should never have spaces.

I wasn't sure where to put the documentation saying this applies to any TXT record... I added it in the Argument Reference section. I felt that if I put it under the SPF record example, people using TXT field not for SPF probably wouldn't check the documentation in this section...

Another option is to add a `~> **Note:**` block at the top before the examples. Do you think this would be preferable instead of the documentation I put in the Argument Reference?